tests/code/test_event_list.py

```diff
@@ -86,3 +86,60 @@ def test_event_list():
         assert Studentled not in oneTimeEvents
 
         transaction.rollback()
+'''
+@pytest.mark.integration

```

Comment:
I'd add a comment about what this fixture does, so it is clear for people unfamiliar with pytest.